public/app/features/admin/UserAdminPage.tsx

```diff
@@ -105,6 +105,8 @@ export class UserAdminPage extends PureComponent<Props> {
     const isLDAPUser = user?.isExternal && user?.authLabels?.includes('LDAP');
     const canReadSessions = contextSrv.hasPermission(AccessControlAction.UsersAuthTokenList);
     const canReadLDAPStatus = contextSrv.hasPermission(AccessControlAction.LDAPStatusRead);
+    const authSource = user?.authLabels?.length && user?.authLabels[0];
```

Comment:
Nit: I guess the following would do the same:
```suggestion
    const authSource = user?.authLabels?.[0]
```